src/components/ModuleCard/ModuleCard.tsx

```diff
@@ -26,7 +27,10 @@ class ModuleCardComponent extends Component<ModuleCardProps > {
       <Card className={Styles.card}>
         {/*<Card.Img variant='top' src={TestImage}/>*/}
         <Card.Body>
-          <Card.Title>{module.name}</Card.Title>
+          {module.disabled
+            ? <Card.Title>{module.name} (disabled)</Card.Title>
+            : <Card.Title>{module.name}</Card.Title>
+          }
```

Comment:
Reduce duplication in your turnery.
```suggestion
          <Card.Title>{module.name} {(module.disabled ? "(disabled)" : "")}</Card.Title>
```